gpu/gpu_linux.go

```diff
@@ -43,10 +43,26 @@ var OneapiGlobs = []string{
 	"/usr/lib*/libze_intel_gpu.so*",
 }
 
+var VulkanGlobs = []string{
+	"/usr/lib/x86_64-linux-gnu/libvulkan.so*",
+	"/usr/lib*/libvulkan.so*",
+}
+
+var capLinuxGlobs = []string{
+	"/usr/lib/x86_64-linux-gnu/libcap.so*",
```

Comment:
adding * after /user/lib also detects 32bit libraries in the system.  Not sure if you want this.

Reply:
Same comment as above, regarding `x86_64` specific usage, this doesn't work on `aarch64` like my Raspberry Pi :)